src/TLuaInterpreter.cpp

```diff
@@ -92,12 +92,15 @@ int speechState = QTextToSpeech::Ready;
 QString speechCurrent;
 #endif // QT_TEXTTOSPEECH_LIB
 
+// No documentation available in wiki - internal function
 static bool isMain(const QString& name)
 {
-    if (name.isEmpty())
+    if (name.isEmpty()) {
         return true;
-    if (name.compare(QStringLiteral("main"), Qt::CaseSensitive) == 0)
+    }
+    if (!name.compare(QStringLiteral("main"))) {
         return true;
```

Comment:
This is a new antipattern, if something return true else return false.
Why not just return something?

Reply:
Mainly because QString::compare is zero when it matches, which (if you're testing for equality) is also an antipattern, burned into the brains of every C programmer for the last half century. That doesn't make it any more correct, thus I really try to avoid directly returning that.

Translated into plain English, "if it's empty, return True. If it's `main`, return True. Otherwise return False" is immediately understandable. "if it's empty, return True, otherwise return whether it … umm … equals `main`" somewhat less so.

Alternately of course we could rewrite that test as `name == "main"` …